pkg/config/setup/config.go

```diff
@@ -1020,6 +1020,10 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.SetKnown("reverse_dns_enrichment.rate_limiter.throttle_error_threshold")
 	config.SetKnown("reverse_dns_enrichment.rate_limiter.recovery_intervals")
 	config.BindEnvAndSetDefault("reverse_dns_enrichment.rate_limiter.recovery_interval", time.Duration(0))
+
+	// Default payloads (disabled for Error Tracking Standalone, Logs Collection Only)
+	config.BindEnvAndSetDefault("default_payloads.enabled", true)
```

Comment:
Could we be more specific in the naming ? What about `infra_payloads` ?

Also, are we planning more setting in this section ? If now what about `infra_payloads_enabled` or something similar ?

Reply:
I think it was more following the pattern other configs have. Agree that it should be something with `infra` in the name (I'm still voting for something like `infra_monitoring`)